src/chaos.js

```diff
@@ -52,4 +91,64 @@ export class KubernetesClient {
         this.kubernetes.namespaces.delete(randomNamespace.name)
         return randomNamespace.name
     }
-}
\ No newline at end of file
+}
+
+export class StressNodesAttack {
+    constructor(client) {
+        this.kubernetes = client
+        this.name = 'stress-nodes'
+        this.namespace = 'default'
+        this.cpu = '0'
+        this.io = '0'
+        this.vm = '0'
+        this.vmBytes = '256MB'
+        this.timeout = '10'
+    }
+
+    inNamespace(namespace) {
+        this.namespace = namespace
+        return this
+    }
+    inNodes(nodes) {
+        this.nodes = nodes
+        return this
+    }
+    withName(name) {
+        this.name = name
+        return this
+    }
+    withCPU(cpu) {
+        this.cpu = cpu
+        return this
+    }
+    withIO(io) {
+        this.io = io
+        return this
+    }
+    withVM(vm) {
+        this.vm = vm
+        return this
+    }
+    withVMBytes(vmBytes) {
+        this.vmBytes = vmBytes
+        return this
+    }
+    withTimeout(timeout) {
+        this.timeout = timeout
+        return this
+    }
+    start() {
```

Comment:
I think `run` communicates better what this method does. At least for me, `start` refers to something that later must be stopped. Also, I don't see any reason for not running multiple times the test. But I recon this is a matter of taste.